Runner.php

```diff
@@ -122,7 +122,7 @@ public function run($files)
             return $resultFile;
         } catch (Exception $e) {
             error_log("Exception: " . $e->getMessage() . " in " . $e->getFile() . "\n" . $e->getTraceAsString());
-            return false;
+            return "cc_exception_thrown";
```

Comment:
(Assuming this is possible) WDYT about returning `$e` then checking (PHP's version of) `is_a?(Exception)`